### PR TITLE
DEV2-3078: Add chat services to helm

### PR DESCRIPTION
**Pull request by assapir**
Everything is on. No need to enable the ability to turn things off for now.
I think it should be bumped to 4.7.0 - as it will be a new release, no?

### CODE REVIEW

**Comment by yanir-codota** [comment]
> chat is **off** by default. I think it should be bumped to 4.7.0 - as it will be a new release, no?

definitely. It should not under any circumstances be part of this 4.6.0 release.

File: charts/tabnine-cloud/values.yaml [hunk 1]
```diff
@@ -49,9 +49,42 @@ global:
...
+      repository: chat
+      tag: v3
+    nodeSelector: {}
+    service:
+      type: ClusterIP
+      port: 80
+      annotations: {}
+    env: {}
+    volumes: []
+    volumeMounts: []
```
Comment (SimSimY) [inline charts/tabnine-cloud/values.yaml:85]: are thoose in use (by the template)? I don't see them in the deployment

**Review by SimSimY — APPROVED** [review]
Looks good + check if all (new) params in values.yaml are used by the template

File: charts/tabnine-cloud/values.yaml (hunk 1 above)
Comment (assapir) [reply charts/tabnine-cloud/values.yaml:85]: good catch!

**Review by SimSimY — APPROVED** [review]
Looks good + check if all (new) params in values.yaml are used by the template

**Pull request merged**